pom.xml

```diff
@@ -12,7 +12,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>za.co.absa</groupId>
 	<artifactId>abris_2.11</artifactId>
-	<version>2.1.0</version>
+	<version>2.2.0-SNAPSHOT</version>
```

Comment:
Why `-SNAPSHOT`? I think if you merge to master, it's basically a release, so it shouldn't be a snapshot, no?